src/VirtualClient/VirtualClient.Actions/Memtier/MemtierBenchmarkClientExecutor.cs

```diff
@@ -311,8 +346,15 @@ private Task ExecuteWorkloadsAsync(IPAddress serverIPAddress, ServerState server
                             // memtier_benchmark Documentation:
                             // https://github.com/RedisLabs/memtier_benchmark
 
-                            string commandArguments = commandArguments = $"--server {serverIPAddress} --port {serverPort} {this.CommandLine}";
-                            
+                            if (string.Equals(this.IsTLSEnabled, "yes", StringComparison.OrdinalIgnoreCase))
```

Comment:
As mentioned above, let's go back to IsTLSEnabled being a proper boolean data type.